ui/index.jsp

```diff
@@ -1761,73 +1761,73 @@
-        <script type="text/javascript" src="scripts/installWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/installWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/projects.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/dashboard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/instanceWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/instanceWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/affinity.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/affinity.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/instances.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/events.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/regions.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/regions.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/ipRules.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/enableStaticNAT.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/securityRules.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/vpc.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/vpc.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/network.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/recurringSnapshots.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/uploadVolume.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/storage.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/templates.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/accountsWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/accountsWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/accounts.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/configuration.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/globalSettings.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/zoneWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/physicalResources.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/ui-custom/zoneWizard.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/system.js?t=<%=now%>"></script>
-        <script type="text/javascript" src="scripts/domains.js?t=<%=now%>"></script>
```

Comment:
There shouldn't be any cache issues using any of the current browsers (by current i mean since a few years back) with default settings.
If the browser is specifically configured by the user to don't look for updates, the user should just do a ctrl+f5. In normal scenario, browser will always ask for the files and tomcat will respond with HTTP 304 (not modified) unless it actually is modified.. so it shouldn't affect the UI developers.
There are tons of browser settings changes that would break the site completely, disabling static file updates lookup is far from being the worse setting a user can change from the defaults on any browser :)
We should not be hitting everyone's performance badly just to support a scenario where a user changes a browser setting that provokes this specific behavior (on any site).
Any way.. if it that was ever an issue... it would have not been fixed by placing the timestamp... since the timestamp is not on all .js files (just most of them) in index.jsp :)